autoform-inputs.js

```diff
@@ -325,8 +325,15 @@ updateAllTrackedFieldValues = function updateAllTrackedFieldValues(formId) {
 
 getAllFieldsInForm = function getAllFieldsInForm(template) {
   // Get all elements with `data-schema-key` attribute, unless disabled
-  return template.$("[data-schema-key]").not("[disabled]");
+  // return template.$("[data-schema-key]").not("[disabled]");
   // Exclude fields in sub-forms, since they will belong to a different AutoForm and schema.
   // TODO need some selector/filter that actually works correctly for excluding subforms
-  // return template.$('[data-schema-key]').not("[disabled]").not(template.$('form form [data-schema-key]'));
+  return template.$('[data-schema-key]').not("[disabled]").not(template.$('form form [data-schema-key]'));
```

Comment:
No because when called for the subform, `template.$('form form [data-schema-key]')` will match every subform input. Because in the DOM they all have two form element ancestors.